### PR TITLE
[New] update backend packages

### PR DESCRIPTION
**Pull request by mo-5**
#### Checklist
If the database has been modified:
- [ ] Update database diagram 
- [ ] Create new migration 

If API endpoints has been modified:
- [ ] Update backend documentation 
- [ ] Create/Update validators for the API endpoints 
- [ ] The API endpoints are properly secured with keycloak 

If the UI has been modified:
- [ ] It is translated in both language (with no hard coded text) 
- [ ] Has been tested on IE
- [ ] The modifications are tab friendly
- [ ] It is accessible

### CODE REVIEW

File: services/backend/src/core/keycloak/keycloak.js [hunk 1]
```diff
@@ -34,12 +34,10 @@ const getGroupIds = async (accessToken) => {
...
-  const groupIds = groups.data[0].subGroups.map(({ id, name }) => {
-    return {
-      id,
-      name,
-    };
-  });
+  const groupIds = groups.data[0].subGroups.map(({ id, name }) => ({
+    id,
+    name,
+  }));
```
Comment (dynamic11) [inline services/backend/src/core/keycloak/keycloak.js:40]: is this happening because of the eslint update? I don't mind it, its cleaner but I'm curious

**Review by dynamic11 — APPROVED** [review]
I played around and everything seems to be working

File: services/backend/src/core/keycloak/keycloak.js (hunk 1 above)
Comment (mo-5) [reply services/backend/src/core/keycloak/keycloak.js:40]: Yeah its from the eslint update

**Comment by mo-5** [comment]
I am not sure why its failing the tests. I will look into it

**Comment by dynamic11** [comment]
everything is failing unit tests. its not just you. I'm not sure when this issue was introduced but im looking into it

**Pull request merged**